File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.cpp
// Last updated: 4/8/2026, 5:08:46 PM
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        for(int i = 1; i < nums.size(); i++) {
            nums[i] += nums[i-1];
        }
        return nums;
    }
};

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 4/8/2026, 5:08:45 PM
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = [nums[0]]
        for i in range(1, len(nums)):
            result.append(result[-1] + nums[i])
        return result